src/App.js

```diff
@@ -29,37 +29,94 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
-
+  // const [winner, setWinner] = useState('');
+  const [currentPlayer, setCurrentPlayer] = useState(Player1);
   // Wave 2
   // You will need to create a method to change the square
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
 
+  const onClickSquare = (id) => {
+    console.log(id);
```

Comment:
When pushing code to be reviewed, we should eliminate any print statements or console logs.

```suggestion

```